<launch>
    <node pkg="camera1394" type="camera1394_node" name="videre_cam">
        <!-- <param name="guid" value="00b09d0100a80b41" /> -->
        <param name="video_mode" value="640x480_mono8" />
        <param name="iso_speed" value="400" />
        <param name="frame_rate" value="60" />
        <!-- <param name="bayer_pattern" value="gbgr" /> -->
        <!-- <param name="bayer_method" value="" /> -->
        <!-- <param name="camera_info_url" value="package://videre_cam_interface/cam1.yaml" /> -->

        <remap from="camera/image_raw" to="cam1/image_raw" />
    </node>

    <node pkg="videre_cam_interface" type="extract_lines" name="cam1">
        <param name="image_raw" value="/cam1/image_raw"/>
        <param name="image_clean" value="/cam1/image_clean"/>
    </node>

</launch>